test/e2e/fixtures/e2e_suite.go

```diff
@@ -73,13 +73,25 @@ func (s *E2ESuite) TearDownSuite() {
 func (s *E2ESuite) BeforeTest(_, _ string) {
 	s.Diagnostics = &Diagnostics{}
 
+	// delete all cron workflows
+	cronList, err := s.cronClient.List(metav1.ListOptions{LabelSelector: label})
```

Comment:
let deletes crons before workflows, in case one get scheduled

Reply:
Nice catch here